### ISSUE TITLE
SubmitGem.java:39-41: Gem submission implementation. The...

### ISSUE THREAD

**Issue by 0pdd** [artipie/gem-adapter#9]
The puzzle `32-7ef023e4` from #32 has to be resolved: 

https://github.com/artipie/gem-adapter/blob/1b2c6cf613f1ef3e226304e53b823d83721763ea/src/main/java/com/artipie/gem/SubmitGem.java#L39-L41

The puzzle was created by @Sammers21 on 27-Mar-20. 

Estimate: 120 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@2cec44a3fa98, thread: pool-503-thread-1
com.zerocracy.radars.github.ThrottledComments[74] java.lang.IllegalStateException: Can't post an identical message to artipie/gem-adapter#9: @g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue
```

[0.52.2](https://github.com/zerocracy/farm): Issue: artipie/gem-adapter#9, Action: `opened`

**Comment by g4s8** [comment]
@rultor release, tag=`0.1.3`

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#24](https://github.com/artipie/gem-adapter/issues/24), [#25](https://github.com/artipie/gem-adapter/issues/25) are still not solved.

**Comment by 0pdd** [comment]
@0pdd the puzzle [#25](https://github.com/artipie/gem-adapter/issues/25) is still not solved; solved: [#24](https://github.com/artipie/gem-adapter/issues/24).

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#25](https://github.com/artipie/gem-adapter/issues/25), [#30](https://github.com/artipie/gem-adapter/issues/30) are still not solved; solved: [#24](https://github.com/artipie/gem-adapter/issues/24).

**Comment by 0pdd** [comment]
The puzzle `32-7ef023e4` has disappeared from the source code, that's why I closed this issue.

**Comment by 0pdd** [comment]
@0pdd all 3 puzzles are solved here: [#24](https://github.com/artipie/gem-adapter/issues/24), [#25](https://github.com/artipie/gem-adapter/issues/25), [#30](https://github.com/artipie/gem-adapter/issues/30).

**Comment by 0crat** [comment]
Job `gh:artipie/gem-adapter#9` is not assigned, can't get performer

**Comment by 0pdd** [comment]
@0pdd the puzzle [#42](https://github.com/artipie/gem-adapter/issues/42) is still not solved; solved: [#24](https://github.com/artipie/gem-adapter/issues/24), [#25](https://github.com/artipie/gem-adapter/issues/25), [#30](https://github.com/artipie/gem-adapter/issues/30).

**Comment by 0pdd** [comment]
@0pdd all 4 puzzles are solved here: [#24](https://github.com/artipie/gem-adapter/issues/24), [#25](https://github.com/artipie/gem-adapter/issues/25), [#30](https://github.com/artipie/gem-adapter/issues/30), [#42](https://github.com/artipie/gem-adapter/issues/42).
